switch/ea4tx_remotebox/remotebox.go

```diff
@@ -207,14 +207,14 @@ func (r *Remotebox) Init() error {
 		StopBits:    1,
 	}
 
-	if r.rb_Connection == 0 { // serial
+	if r.Connection == 0 { // serial
```

Comment:
I suggest to introduce a custom type for `connection` - for example `rbConnection`. As done with all consts of type `rbModel`, you can auto generate a dictionary holding the possible values with go generate after adding it to the top line of the file.
Instead of using an `if` clause, the code will get easier extensible using a `switch` clause. 

You can check if the string matches any existing values of `rbConnection` in the `parse_ea4tx_remotebox_config` file.